app/policies/user_policy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class UserPolicy < ApplicationPolicy
+  def create?
+    admin_only
+  end
```

Comment:
Does that break user registration (school induction coordinators in particular)?

Reply:
No, since we don't call `authorize` there. If we did, then yes, it would

Reply:
I'm expect to have several base controllers which all mandate after_action :verify_authorized & after_action :verify_policy_scoped for different site areas, but that the registration stuff will not inherit from them